.rubocop.yml

```diff
@@ -0,0 +1,45 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+
```

Comment:
can you  add a comment about why we're disabling those? (most of them seems useful)

Reply:
We disabled the rules that generates a lot of linting issues to reduce the linting noise and we plan to clean up the code later as requested in #37

Reply:
#37 sounds unrelated to lintint, are you referring to #14?

Reply:
Yes, referring to #14

Reply:
This PR is marked as fixing #14, so unless we keep #14 open after merging it we should make sure we have another issue to follow up with more lintint changes.

I would recommend enabling those checks and addressing them here for the dualgate rules

Reply:
@proppy Let's move forward with the merge. I have opened new issue #40 to keep track of this required change.